dio/dio.php

```diff
@@ -12,117 +12,160 @@
- * @param int $cmd The file descriptor returned by dio_open().
- * @param mixed ...$args args is an associative array, when cmd is F_SETLK or F_SETLLW, with the following keys:
- * start - offset where lock begins
- * length - size of locked area. zero means to end of file
- * whence - Where l_start is relative to: can be SEEK_SET, SEEK_END and SEEK_CUR
- * type - type of lock: can be F_RDLCK (read lock), F_WRLCK (write lock) or F_UNLCK (unlock)
+ * @param resource $fd
+ * The file descriptor returned by dio_open().
+ * @param int $cmd
+ * The file descriptor returned by dio_open().
+ * @param mixed ...$args
+ * args is an associative array, when cmd is **F_SETLK** or **F_SETLLW**, with the following keys:
+ * - *start* - offset where lock begins
+ * - *length* - size of locked area. zero means to end of file
+ * - *whence* - Where l_start is relative to: can be **SEEK_SET**, **SEEK_END** and **SEEK_CUR**
+ * - *type* - type of lock: can be **F_RDLCK** (read lock), **F_WRLCK** (write lock) or **F_UNLCK** (unlock)
  * @return mixed Returns the result of the C call.
  */
 function dio_fcntl($fd, int $cmd, ...$args) {}
 
 /**
+ * Opens a file (creating it if necessary) at a lower level than theC library input/ouput stream functions allow
+ *
  * dio_open ( string $filename , int $flags [, int $mode = 0 ] ) : resource
- * The function dio_close() closes the file descriptor fd.
+ *
  * @link https://www.php.net/manual/en/function.dio-open.php
  * @param string $filename The pathname of the file to open.
  * @param int $flags
- * The flags parameter is a bitwise-ORed value comprising flags from the following list. This value must include one of O_RDONLY, O_WRONLY, or O_RDWR. Additionally, it may include any combination of the other flags from this list.
- * O_RDONLY - opens the file for read access.
- * O_WRONLY - opens the file for write access.
- * O_RDWR - opens the file for both reading and writing.
- * O_CREAT - creates the file, if it doesn't already exist.
- * O_EXCL - if both O_CREAT and O_EXCL are set and the file already exists, dio_open() will fail.
- * O_TRUNC - if the file exists and is opened for write access, the file will be truncated to zero length.
- * O_APPEND - write operations write data at the end of the file.
- * O_NONBLOCK - sets non blocking mode.
- * O_NOCTTY - prevent the OS from assigning the opened file as the process's controlling terminal when opening a TTY device file.
- * @param int $mode If flags contains O_CREAT, mode will set the permissions of the file (creation permissions). mode is required for correct operation when O_CREAT is specified in flags and is ignored otherwise.
+ * The flags parameter is a bitwise-ORed value comprising flags from the following list.
+ * This value must include one of **O_RDONLY**, **O_WRONLY**, or **O_RDWR**.
+ * Additionally, it may include any combination of the other flags from this list.
+ * - **O_RDONLY** - opens the file for read access.
```

Comment:
Resulting formatting of these changes doesn't look correct, unfortunately: 
![image](https://user-images.githubusercontent.com/33625946/157683457-f7c26c50-d208-489f-b2c1-422f0dcebded.png)

Please fix or just leave as it is, since I don't see any benefits in these changes.